integration_tests/src/main/python/cast_test.py

```diff
@@ -737,7 +850,14 @@ def test_cast_fallback_not_UTC(from_gen, to_type):
         {"spark.sql.session.timeZone": "+08",
          "spark.rapids.sql.castStringToTimestamp.enabled": "true"})
 
-def test_cast_date_integral_and_fp():
+
+def test_cast_date_integral_and_fp_ansi_off():
+    """
+    This tests that a date column can be cast to different numeric/floating-point types.
+    This needs to be tested with ANSI disabled, because some of these conversions are
```

Comment:
> because some of these conversions are not ANSI-compliant

Are these specific source/target types that we can move to a dedicated test? Can you elaborate about "some"?

Reply:
I just went through all the types.  Not one conversion is ANSI compliant.  Apropos, I've updated the test documentation to reflect this.